packages/opentelemetry-cloud-monitoring-exporter/README.md

```diff
@@ -43,4 +44,6 @@ With the above you should now be able to navigate to the Google Cloud Monitoring
 - Learn more about Google Cloud Monitoring at https://cloud.google.com/monitoring
 
 [license-url]: https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/blob/master/LICENSE
+[npm-url]: https://www.npmjs.com/package/@google-cloud/opentelemetry-cloud-trace-exporter
+[npm-img]: https://badge.fury.io/js/%40google-cloud%2Fopentelemetry-cloud-trace-exporter.svg
```

Comment:
same here, opentelemetry-cloud-trace-exporter to opentelemetry-cloud-monitoring-exporter.